web/admin/application/configs/klear/FixedCostsRelInvoiceSchedulersList.yaml

```diff
@@ -0,0 +1,82 @@
+#include conf.d/mapperList.yaml
+#include conf.d/actions.yaml
+
+
+production:
+  main:
+    module: klearMatrix
+    defaultScreen: fixedCostsRelInvoiceSchedulersList_screen
+  screens: &fixedCostsRelInvoiceSchedulers_screensLink
+    fixedCostsRelInvoiceSchedulersList_screen: &fixedCostsRelInvoiceSchedulersList_screenLink
+      controller: list
+      class: ui-silk-money-add
+      pagination:
+        items: 25
+      <<: *FixedCostsRelInvoiceSchedulers
+      title: _("List of %s %2s", ngettext('Fixed cost', 'Fixed costs', 0), "[format| (%parent%)]")
+      forcedValues:
+        <<: *forcedBrand
+      fields: &fixedCostsRelInvoiceSchedulers_fieldsLink
+        options:
+          title: _("Options")
+          screens:
+            fixedCostsRelInvoiceSchedulersEdit_screen: true
+          dialogs:
+            fixedCostsRelInvoiceSchedulersDel_dialog: true
+          default: fixedCostsRelInvoiceSchedulersEdit_screen
+      options:
+        title: _("Options")
+        screens:
+          fixedCostsRelInvoiceSchedulersNew_screen: true
+        dialogs:
+          fixedCostsRelInvoiceSchedulersDel_dialog: true
+    fixedCostsRelInvoiceSchedulersNew_screen: &fixedCostsRelInvoiceSchedulersNew_screenLink
+      <<: *FixedCostsRelInvoiceSchedulers
+      controller: new
+      class: ui-silk-add
+      label: true
+      multiInstance: true
+      title: _("Add %s", ngettext('Fixed cost', 'Fixed costs', 1), "[format| (%parent%)]")
+      shortcutOption: N
+      fullWidth: true
+      filterField: invoiceScheduler
```

Comment:
We usually set the filterField in the _parent_ screen, but if you want to add in the _child_ one, you can also add it to the List screen and remove it from _InvoiceSchedulersList.yaml_

Reply:
Done, thanks :)